src/components/Form.js

```diff
@@ -0,0 +1,16 @@
+const Form = () => {
+  return (
+    <div className="form-section">
+    <h4>ADD NEW BOOK</h4>
+    <form>
+      <input type='text' placeholder='book title'/>
+      <select id="cars" name="cars">
+        <option value="volvo">Author</option>
+      </select>
+      <button className="form-btn">Add book</button>
+    </form>
```

Comment:
✅ Renders a form with inputs for the title 👍🏻 
✅ Has a button to submit 👍🏻 

- [ ] [REQUIRED] Kindly add an input for the author of the book